Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM python:3.8-slim AS conpot-builder
+
+# Install dependencies
+#RUN apt update \
+#    && apt install ipmitool git python3-dev build-base py-cffi libxslt-dev libffi-dev py3-pip \
+#    && apt install mysqlclient-dev
+
+RUN apt-get update && apt-get install -y \
+    libmariadb-dev \
+    gcc \
+    && rm -rf /var/lib/apt/lists/*
+
+# Copy the app from the host folder (probably a cloned repo) to the container
+RUN adduser --disabled-password --gecos "" conpot
+
+RUN mkdir /opt/conpot
+COPY . /opt/conpot/
+RUN chown conpot:conpot -R /opt/conpot
+
+# Install Python requirements
+USER conpot
+
+WORKDIR /opt/conpot
+
+ENV PATH=$PATH:/home/conpot/.local/bin
+
+RUN pip3 install --user --no-cache-dir -r requirements.txt
+
+# Install the Conpot application
+RUN python3 setup.py install --user --prefix=
+
+
+# Run container
+FROM python:3.8-slim
+
+RUN apt-get update && apt-get install -y \
+    wget \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN adduser --disabled-password --gecos "" conpot
+WORKDIR /home/conpot
+
+COPY --from=conpot-builder --chown=conpot:conpot /home/conpot/.local/ /home/conpot/.local/
+RUN mkdir -p /etc/conpot /var/log/conpot /usr/share/wireshark \
+    && wget https://github.com/wireshark/wireshark/raw/master/manuf -o /usr/share/wireshark/manuf
+
+# Create directories
+RUN mkdir -p /var/log/conpot/ \
+    && mkdir -p /data/tftp/ \
```

Comment:
Don't think this directory is used.

Reply:
It seems to be used here: https://github.com/mushorg/conpot/blob/master/conpot/protocols/tftp/tftp_server.py#L74-L79

Reply:
If the comment in the template is to be believed then it's relative to `conpot/tests/data/data_temp_fs/`. The VFS code is a little too involved for me so I haven't confirmed this.

Not much harm in leaving it in the Dockerfile but it may be a bit puzzling for the next reader. We can also open a separate issue for this. Or maybe @xandfury can shed some light?

https://github.com/mushorg/conpot/blob/0c5f4e1765b3ddeaa0b0704d5c8ca9d69af39e85/conpot/templates/default/tftp/tftp.xml#L2-L3

https://github.com/mushorg/conpot/blob/0c5f4e1765b3ddeaa0b0704d5c8ca9d69af39e85/conpot/core/virtual_fs.py#L56-L64

Reply:
Sorry for being so late to respond. :sweat_smile: 

The VFS must be docker volume. Since the user might need to persist the artefacts collected. But there are multiple ways of going about this. I would keep the directories just in case the user doesn't want the keep the collected artefacts.

@srenfo @glaslos